Formula/unit.rb

```diff
@@ -0,0 +1,95 @@
+class Unit < Formula
+  desc "Dynamic web and application server for multiple languages"
+  homepage "https://unit.nginx.org/"
+  url "https://unit.nginx.org/download/unit-1.1.tar.gz"
+  sha256 "7c66365f5ea87e7e8903bac9d639676947fe2ab7aa799ce35defc426b3409ee0"
+  head "https://hg.nginx.org/unit", :using => :hg
+
+  option "with-debug", "Build with debug logging enabled"
+
+  depends_on "openssl"
+  depends_on "pcre"
+
+  depends_on "go" => :optional
+  depends_on "perl" => :optional
+  depends_on "python" => :optional
+  depends_on "ruby" => :optional
+
+  def install
+    openssl = Formula["openssl"]
+    pcre = Formula["pcre"]
+
+    configure_args = [
+      "--prefix=#{prefix}",
+      "--bindir=#{bin}",
+      "--sbindir=#{bin}",
+      "--modules=#{lib}/unit",
```

Comment:
What would be the default value of `modules`, `state`,  `log`, `pid` and `control`?

Reply:
Covered above.